### ISSUE TITLE
JOSS manuscript clean up [2]

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spopt#248]
xref #246, openjournals/joss-reviews#3330

* [x] clean duplicated URLs in references
* [x] figure 2 caption URL
* [x] code block for Figure 2
   * format for reduced width, line length coming off page
   * possibly reduce figure size?
* [x] clean invalid DOIs

**Comment by jGaboardi** [comment]
I think the code block in Figure 2 is too large are cumbersome (see current rendering [here](https://github.com/openjournals/joss-papers/blob/joss.03330/joss.03330/10.21105.joss.03330.pdf)). Moreover, it can't be run by "copy&paste"ing the code and creates half-blank page situation in the typesetting. I vote that we remove that code block and point readers to the notebook, which we are already doing in the caption. What are your thoughts @gegen07?

**Comment by gegen07** [comment]
TL;DR: I think it's ok to remove the mentioned code block. 

When I read a paper, code blocks are, generally, hard to understand and it eventually makes me disinterested. As the tutorial aims at the code explanation and how API works, I think it's ok to remove the code block.

**Comment by jGaboardi** [comment]
@gegen07, I also think we could add the following sending to the facility location section:

> Moreover, we have included functionality to place pre-determined site locations within the facility selection pool, allowing for realistic scenarios whereby 1 or more *new* facilities can be added to an *existing* set of locations.

What do you think?

**Comment by gegen07** [comment]
That's awesome. I think we can also explicitly add that this is a feature which no of the open source tools provide. What do you think?
